api_spec.yaml

```diff
@@ -115,6 +115,7 @@ paths:
       operationId: revokeOAuth2AuthorizationCode
       description: |
         Revoke all authorization codes belonging to a session or to a client (across all tenants).
+        [Click here](https://app.code2flow.com/gPxFBmp7sdCR) for flow diagram.
```

Comment:
also add queries for checking if session handle or client id exist so appropriate error can be returned

Reply:
changes made